core/src/main/java/tachyon/worker/TachyonWorker.java

```diff
@@ -4,9 +4,9 @@
  * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
  * "License"); you may not use this file except in compliance with the License. You may obtain a
  * copy of the License at
- *
+ * 
```

Comment:
It is a little confusing to me. I checked other files. Almost half of the files have the trailing whitespace here and others do not. It is not consistent. The formatter will also add the whitespace here.